R/responsibilities.R

```diff
@@ -0,0 +1,318 @@
+helper_touchstones <- function(modelling_group_id, location = NULL) {
+  assert_scalar_character(modelling_group_id)
+  path <- sprintf("/modelling-groups/%s/responsibilities/", modelling_group_id)
+  montagu_api_GET(location, path)
+}
+
+##' This may get a name change pending splitting apart of various
```

Comment:
Throughout, you've got one of the stub bits of documentation that are my fault, I think,  Can we replace this with a proper sentence in each case?

Reply:
Replaced each with some more descriptive text...